addons/sale/models/sale_order_line.py

```diff
@@ -435,7 +435,7 @@ def _compute_price_unit(self):
         for line in self:
             # check if there is already invoiced amount. if so, the price shouldn't change as it might have been
             # manually edited
-            if line.qty_invoiced > 0:
+            if line.qty_invoiced > 0 or line.is_expense:
```

Comment:
it makes sense if the expense_policy of product is at_cost, but if the value was sales_price it would have been ok to have it updated I think

not doing it would cause eg. pricelist based on number of product / date to not be applied when changing the line. so maybe we should only do this if `line.product_id.expense_policy == 'at_cost'`